llvm/include/llvm/Cheerp/MemoryInit.h

```diff
@@ -0,0 +1,31 @@
+//===-- MemoryInit.h - Cheerp helper -------------------------===//
```

Comment:
I know that we are guilty of this in many headers, but here instead of just "cheerp helper", put a one liner explaining what this does.